python/chronos/use-case/pytorch-forecasting/TFT/tft.py

```diff
@@ -0,0 +1,170 @@
+    time_varying_known_reals=["time_idx", "price_regular", "discount_in_percent"],
+    time_varying_unknown_categoricals=[],
+    time_varying_unknown_reals=[
+        "volume",
+        "log_volume",
+        "industry_volume",
+        "soda_volume",
+        "avg_max_temp",
+        "avg_volume_by_agency",
+        "avg_volume_by_sku",
+    ],
+    target_normalizer=GroupNormalizer(
+        groups=["agency", "sku"], transformation="softplus"
+    ),  # use softplus and normalize by group
+    add_relative_time_idx=True,
+    add_target_scales=True,
+    add_encoder_length=True,
+)
+
+# create validation set (predict=True) which means to predict the last max_prediction_length points in time
+# for each series
+validation = TimeSeriesDataSet.from_dataset(training, data, predict=True, stop_randomization=True)
+
+# create dataloaders for model
+batch_size = 128  # set this between 32 to 128
+train_dataloader = training.to_dataloader(train=True, batch_size=batch_size, num_workers=0)
+val_dataloader = validation.to_dataloader(train=False, batch_size=batch_size * 10, num_workers=0)
+# calculate baseline mean absolute error, i.e. predict next value as the last available value from the history
+actuals = torch.cat([y for x, (y, weight) in iter(val_dataloader)])
+baseline_predictions = Baseline().predict(val_dataloader)
+(actuals - baseline_predictions).abs().mean().item()
+# configure network and trainer
+pl.seed_everything(42)
+trainer = Trainer(
+    gpus=0,
+    # clipping gradients is a hyperparameter and important to prevent divergance
+    # of the gradient for recurrent neural networks
+    gradient_clip_val=0.1,
+)
+
+
+tft = TemporalFusionTransformer.from_dataset(
+    training,
+    # not meaningful for finding the learning rate but otherwise very important
+    learning_rate=0.03,
+    hidden_size=16,  # most important hyperparameter apart from learning rate
+    # number of attention heads. Set to up to 4 for large datasets
+    attention_head_size=1,
+    dropout=0.1,  # between 0.1 and 0.3 are good values
+    hidden_continuous_size=8,  # set to <= hidden_size
+    output_size=7,  # 7 quantiles by default
+    loss=QuantileLoss(),
+    # reduce learning rate if no improvement in validation loss after x epochs
+    reduce_on_plateau_patience=4,
+)
+print(f"Number of parameters in network: {tft.size()/1e3:.1f}k")
+# find optimal learning rate
+res = trainer.tuner.lr_find(
+    tft,
+    train_dataloader=train_dataloader,
+    val_dataloaders=val_dataloader,
+    max_lr=10.0,
+    min_lr=1e-6,
+)
+
+print(f"suggested learning rate: {res.suggestion()}")
+
+# configure network and trainer
+early_stop_callback = EarlyStopping(monitor="val_loss", min_delta=1e-4, patience=10, verbose=False, mode="min")
+lr_logger = LearningRateMonitor()  # log the learning rate
+logger = TensorBoardLogger("lightning_logs")  # logging results to a tensorboard
+
+trainer = Trainer(
+    max_epochs=30,
```

Comment:
Oh, could you paste the error message/stack you met here when you set `num_processes>1`